src/scrapeconfig.ts

```diff
@@ -58,8 +59,8 @@ type ScrapeConfigOptions = {
   proxy_pool?: string;
   session?: string;
   tags?: string[];
-  format?: Format;
-  format_options?: FormatOption[];
+  format?: 'json' | 'text' | 'markdown' | 'clean_html' | 'raw' | Format;
+  format_options?: ('no_links' | 'no_images' | 'only_content')[] | FormatOption[];
```

Comment:
```
('no_links' | 'no_images' | 'only_content' | FormatOption)[];
```
This will allow mixing of Enum and string options and since we allow both we should allow mixing of them as well.

Reply:
Mixing between Enums and string options is already allowed. For example: `format_options: ['no_images', 'only_content', ScrapeFormatOptions.NO_LINKS]`, or am I missing something?

Reply:
yes but the `('no_links' | 'no_images' | 'only_content')[] | FormatOption[]` type here reads as `Array of string literals OR Array of Enums` which doesn't allow mixing. It should read as `Array of string literals or Enums` which would be:
```
('no_links' | 'no_images' | 'only_content' | FormatOption)[];
```